src/bsp/src/stm32/DefaultSettings.cmake

```diff
@@ -7,6 +7,10 @@ if (${COMPILE_STM32_F429ZI})
     set(USER_DATA_FLASH_SIZE "128K")
     set(USER_DATA_FLASH_SECTOR "23")
     set(USER_DATA_FLASH_START_ADDRESS "0x081E0000")
+elseif (${COMPILE_STM32_H735ZG})
+    set(USER_DATA_FLASH_SIZE "128K")
+    set(USER_DATA_FLASH_SECTOR "7")
+    set(USER_DATA_FLASH_START_ADDRESS "0x080E0000")
```

Comment:
@cquesnel  Can you double check if that's ok with your confluence?

Reply:
Seems good